lib/RESTClient.js

```diff
@@ -23,6 +24,23 @@ function errorMap(mdError) {
     return map[mdError] ? map[mdError] : mdError;
 }
 
+/**
+ * Turns the bootstrap list into an array of host/port objects.
+ *
+ * @param {String[]} list - the list to process
+ *
+ * @return {{ host: string, port: number }[]}
+ */
+function parseBootstrapList(list) {
+    return list.map(elem => {
+        const tmp = elem.split(':');
+        const host = tmp[0];
+        const port = Number.parseInt(tmp[1], 10);
```

Comment:
Having a default would simplify greatly my work by not having to deal with side projects at first, but I think we should be strict when everything is up-to-date. Implicit configuration is the kind of gotcha we do not want to deal with later on.